Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/VisualThemes/ThemeEngines/ShaderUnityUiColorTheme.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using UnityEngine.UI;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// This class is used to apply Color changes to shader properties on Unity UI based elements, since they do not support MaterialPropertyBlocks. 
+    /// Use this only on a Unity UI based target component.
+    /// </summary>
+    public class ShaderUnityUiColorTheme : ShaderUnityUiTheme
```

Comment:
Nit: feel free to ignore. But shouldn't it be ShaderUnityUIColorTheme? with the i capitolized?
https://docs.unity3d.com/Manual/UISystem.html